librz/include/rz_analysis.h

```diff
@@ -257,6 +257,7 @@ typedef struct rz_analysis_function_t {
 	bool bp_frame;
 	bool is_noreturn; // true if function does not return
 	ut8 *fingerprint; // TODO: make is fuzzy and smarter
+	int argnum; // number of arguments;
```

Comment:
@ret2libc @thestr4ng3r what do you think about this?

Reply:
:+1: seems fine